src/components/ReviewFullPage/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import ReviewCard from "../ReviewCard";
+import { getReviewById } from "../../utils/apis";
+import "./styles.css";
```

Comment:
CSS is looking good, Dave! 🎨

Reply:
Spent a little bit of time trying to get eveything neat for now, thx ;)